Fix:
gen_sds_floats.c:26:1: error: type specifier missing, defaults to 'int' [-Werror,-Wimplicit-int]
--- mfhdf/hdfimport/gen_sds_floats.c.orig	2020-03-03 11:40:50.000000000 -0600
+++ mfhdf/hdfimport/gen_sds_floats.c	2024-06-18 23:16:23.000000000 -0500
@@ -23,7 +23,7 @@
 #define RANK2          2  /* Number of dimensions of the SDS */
 #define RANK3          3  /* Number of dimensions of the SDS */
 
-main() 
+int main(void)
 {
     /************************* Variable declaration ************************/
 
